### DISCUSSION TITLE
Vite hmr not working in remix project with emotion templete

### DISCUSSION THREAD

**Discussion by ecoskun87** [chakra-ui/chakra-ui#9313] [Q&A] [answered]
hello , i installed a remix project and  added chakra with emotion templete. but vite hmr not updating ui anymore  with chakra with emotion. can somebody help me ?

**Comment by segunadebayo** [comment, marked as answer]
The Remix + Emotion template is leftover v2.

Use the current Vite or Remix setup. https://chakra-ui.com/docs/get-started/frameworks/vite
